### PR TITLE
Update celery to 4.1

### PR DESCRIPTION
**Pull request by hsiaoyi0504**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -19,3 +19,4 @@ selenium==3.5.0
 django-suit==0.2.18
 six>=1.10.0
 django-axes==2.3.3
+django-celery-results
```
Comment (hsiaoyi0504) [inline requirements.txt]: @childers Although I am not quite sure what `Django ORM/Cache` means, I think we don't need this. I compared the counterpart in celery documentation:
- 4.1: http://docs.celeryproject.org/en/v4.1.0/django/first-steps-with-django.html#django-celery-results-using-the-django-orm-cache-as-a-result-backend
- 3.1: http://docs.celeryproject.org/en/3.1/django/first-steps-with-django.html#using-the-django-orm-cache-as-a-result-backend

If we need this, we should install `django-celery` before.

File: i5k/celery.py [hunk 2]
```diff
@@ -16,3 +16,6 @@
 app.config_from_object('django.conf:settings')
 app.autodiscover_tasks(lambda: settings.INSTALLED_APPS)
 
+@app.task(bind=True)
```
Comment (hsiaoyi0504) [inline i5k/celery.py]: Can you point me where the documentation describes the usage of this?

Comment (childers) [reply i5k/celery.py]: http://docs.celeryproject.org/en/latest/django/first-steps-with-django.html#using-celery-with-django

Comment (hsiaoyi0504) [reply i5k/celery.py]: Thanks.

However, I think it is for introduction of celery and is not for the upgrade.
Similar syntax can be found older version of the documentation: http://docs.celeryproject.org/en/3.1/django/first-steps-with-django.html

I think what we do here is "For Django users and others who want to keep uppercase names" section of:
http://docs.celeryproject.org/en/v4.1.0/whatsnew-4.0.html#upgrading-from-celery-3-1

File: requirements.txt (hunk 1 above)
Comment (hsiaoyi0504) [reply requirements.txt]: Oh, I finally figured out that `amqp` result backend can't be used anymore ...
@childers Do you need to run `python manage.py migrate` before start using this?

**Comment by hsiaoyi0504** [comment]
Close, will replace this with another branch.
https://github.com/NAL-i5K/genomics-workspace/tree/celery_update_yi

**Pull request closed without merging**